### PR TITLE
fix: Adds missing words to HTML and fixes order

### PR DESCRIPTION
**Pull request by anthony0030**
# Fix Dictionary

Dictionary: ```HTML```

## Description

Adds missing words to HTML
* wacag
* utma
* utmc
* utmb

Fixes order

## References

https://github.com/streetsidesoftware/cspell-dicts/issues/1897

## Checklist

- [X] By submitting this issue, you agree to follow our
      [Code of Conduct](https://github.com/streetsidesoftware/cspell-dicts/blob/main/CODE_OF_CONDUCT.md)

### CODE REVIEW

File: dictionaries/html/src/html.txt [hunk 1]
```diff
@@ -3001,6 +3004,7 @@ vsupne
 vsupnE
 Vvdash
 vzigzag
+wacag
```
Comment (nschonni) [inline dictionaries/html/src/html.txt:3007]: This is a typo of WCAG, and probably doesn't belong in the HTML dictionary, as mentioned over on https://github.com/streetsidesoftware/cspell-dicts/pull/1933
Unless i'm mistaken on what the word is

Comment (anthony0030) [reply dictionaries/html/src/html.txt]: You are correct, sorry :( I removed it.

**Comment by Jason3S** [comment]
@anthony0030,

I think you are using a different sort locale that puts spaces after letters.

**Comment by Jason3S** [comment]
@anthony0030,

I put the utm* terms in fullstack in #1933

**Comment by anthony0030** [comment]
Hi @Jason3S,
As have added them to a different pull request this one has no value, so I will close it.

**Pull request closed without merging**